src/test/java/guitests/UseGlobalConfigsTest.java

```diff
@@ -80,6 +78,7 @@ public void globalConfigTest() {
         pushKeys(KeyCode.SHIFT, KeyCode.SEMICOLON);
         type("issue");
         pushKeys(KeyCode.ENTER);
+        PlatformEx.waitOnFxThread();
```

Comment:
Hmm, but there is still the possibility that it might fail... Do take note of the error if it happens again, otherwise it will just surface at a later time.

By the way, `type` supports the colon character. You can put the colon directly in the string.